src/components/AddPayment.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import FiniteDateRange from '../interfaces/FiniteDateRange'
+import addYears from '../modules/addYears'
+import Button from './Button'
+import DateRangePicker from './DateRangePicker'
+import NumericInput from './NumericInput'
+
+interface AddPaymentProps {
+  getPaymentInfo: (amount: number, paidDays: FiniteDateRange) => void
```

Comment:
get* functions should return something.
Name it `onSubmit`.
Names like `on*` are perfect for many functions passed as arguments